rules-reviewed/jboss-eap/websphere/WebSphere-other.windup.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset id="websphere-other"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            Provides analysis of WebSphere proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="websphere" versionRange="(10,12]"/>
+        <targetTechnology id="eap" versionRange="[6,)"/>
+        <targetTechnology id="java-ee" versionRange="[6,)"/>
+        <tag>reviewed-2015-06-05</tag>
+        <tag>ibm</tag>
+    </metadata>
+
+
+    <rules>
+        <rule id="websphere-other-6bcvk">
```

Comment:
the rule id's should be updated to meet our naming standards:

-01000
-02000
-03000

...
etc